src/ruby/ext/grpc/rb_grpc_imports.h

```diff
@@ -0,0 +1,399 @@
+/*
+ *
+ * Copyright 2016, Google Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are
+ * met:
+ *
+ *     * Redistributions of source code must retain the above copyright
+ * notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following disclaimer
+ * in the documentation and/or other materials provided with the
+ * distribution.
+ *     * Neither the name of Google Inc. nor the names of its
+ * contributors may be used to endorse or promote products derived from
+ * this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+#ifndef GRPC_RB_GRPC_IMPORTS_H_
+#define GRPC_RB_GRPC_IMPORTS_H_
+
+#include <include/grpc/census.h>
+#include <include/grpc/compression.h>
+#include <include/grpc/grpc.h>
+#include <include/grpc/grpc_security.h>
+
```

Comment:
Alright, I give up. There's something wrong here, and I don't know what.

If one runs the script plugin manually, he'd get the following extract:

```
- {arguments: grpc_byte_buffer_reader *reader, header: include/grpc/impl/codegen/byte_buffer.h,
  name: grpc_byte_buffer_reader_readall, return_type: gpr_slice}
- {arguments: grpc_byte_buffer_reader *reader, header: include/grpc/impl/codegen/byte_buffer.h,
  name: grpc_raw_byte_buffer_from_reader, return_type: grpc_byte_buffer *}
- {arguments: gpr_slice s, header: include/grpc/impl/codegen/slice.h, name: gpr_slice_ref,
  return_type: gpr_slice}
```

These files and functions are clearly not represented in here however. So something is wrong with the way the plugin works in regards to listing the necessary public APIs.

With its name, it should be running after `expand_filegroups.py` however, as we are sorting them by name.

So... I am brain farting too much to figure out what's going on.